README.md

```diff
@@ -27,35 +32,46 @@ roles responsible for building and running apps and operating infrastructure.
 
 ## See it in action
 
-[Crossplane](https://github.com/crossplane/crossplane) is the implementation of the Open Application Model specification for Kubernetes. To get started with an example of the Open Application Model, follow the guide to run applications on [Crossplane](https://crossplane.io/docs/v0.11/getting-started/run-applications.html).
+[OAM Kubernetes Runtime](https://github.com/crossplane/oam-kubernetes-runtime) is the officially maintained OAM plugin for Kubernetes. 
+
+OAM Kubernetes Runtime is a [joint effort](https://cloudblogs.microsoft.com/opensource/2020/05/27/open-application-model-oam-v1alpha2-crossplane/) with [the Crossplane community](https://github.com/crossplane/crossplane). Furthermore, to get started with an example of using OAM to deliver both applications and cloud resources in unified approach, please follow the end-to-end [getting started doc in Crossplane](https://crossplane.io/docs/v0.11/getting-started/run-applications.html).
 
 ## The Specification
 
 The specification convention adopts [Kubernetes Resource Model](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/resource-management.md) which we believe will become the standard interface for the majority of platforms in the future.
 
-[Notational Conventions](notational_convention.md)
-
-  1. [Purpose and Goals](1.purpose_and_goals.md)
-  2. [Overview and Terminology](2.overview_and_terminology.md)
-  3. [Workload Types](3.workload.md)
-  4. [The Component Model](4.component.md)
-  5. [Application Scopes](5.application_scopes.md)
-  6. [Traits](6.traits.md)
-  7. [Application Configuration](7.application_configuration.md)
-  8. [Practical Considerations](8.practical_considerations.md)
-  9. [Design Principles](9.design_principles.md)
+  - [Notational Conventions](notational_convention.md)
+  - [Purpose and Goals](1.purpose_and_goals.md)
+  - [Overview and Terminology](2.overview_and_terminology.md)
+  - API Resource Specification
+    - Application Developers
+      - [Components](4.component.md)
+    - Application Operators
+      - [Application Scopes](5.application_scopes.md)
+      - [Traits](6.traits.md)
+      - [Application Configuration](7.application_configuration.md)
+    - Infrastructure Operators/Platform Builders
+      - [Workload Definition](3.workload.md)
+  - [Practical Considerations](8.practical_considerations.md)
+  - [Design Principles](9.design_principles.md)
 
 
 ## Community
 
+### Versioning
+
+Since July 2020, changes to the specification are versioned according to [Semantic Versioning 2.0](https://semver.org/spec/v2.0.0.html) and described in [its release page](https://github.com/oam-dev/spec/releases). Layout changes are not versioned. Specific implementations of the specification (e.g. OAM Kubernetes Runtime ) should specify which version they implement.
+
+Changes to the change process itself are not currently versioned but may be independently versioned in the future.
```

Comment:
What do you mean "Changes to the change process"?

Reply:
Changes to governance model, to spec review/merge process etc.